mllib/src/test/scala/org/apache/spark/mllib/clustering/LDASuite.scala

```diff
@@ -374,9 +374,14 @@ class LDASuite extends SparkFunSuite with MLlibTestSparkContext {
       .values
       .collect()
 
-    expectedPredictions.zip(actualPredictions).forall { case (expected, actual) =>
-      expected._1 === actual._1 && (expected._2 ~== actual._2 relTol 1E-3D)
+    expectedPredictions.zip(actualPredictions).foreach { case (expected, actual) =>
+      assert(expected._1 === actual._1 && (expected._2 ~== actual._2 relTol 1E-3D))
     }
+
+    val topicsBz = ldaModel.topicDistributions(docs.first())._2.toBreeze.toDenseVector
+    val singlePrediction = (argmax(topicsBz), max(topicsBz))
```

Comment:
Can you please change this test to compare the full topicDistribution for the doc (with the distribution computed using the batch topicDistributions() call), rather than just the most important topic?